streampipes-extensions/streampipes-sinks-brokers-jvm/src/main/java/org/apache/streampipes/sinks/brokers/jvm/BrokersJvmInit.java

```diff
@@ -54,16 +50,12 @@ public SpServiceDefinition provideServiceDefinition() {
             "",
             8096)
         .registerPipelineElements(
-            new KafkaPublishSink(),
             new JmsPublisherSink(),
             new RestSink(),
             new BufferRestPublisherSink(),
             new RabbitMqPublisherSink(),
             new MqttPublisherSink(),
             new WebsocketServerSink(),
-            new PulsarPublisherSink(),
-            new RocketMQPublisherSink(),
```

Comment:
Don't we want them to be part of the brokers collection anymore? Or do we not want to have the additional dependency?

Reply:
Yes, I added these to `extensions-all-iiot` so that we don't need the dependency in the broker module.